_posts/2024-01-10-practical_zephyr_basics.md

```diff
@@ -693,4 +693,4 @@ Finally, have a look at the files in the [accompanying GitHub repository](https:
 
 [zephyr-gh-docker]: https://github.com/zephyrproject-rtos/docker-image
 
--->
\ No newline at end of file
+-->
```

Comment:
Could you revert this removal of the new line?

Reply:
yes, sorry!

Reply:
No problem, thank you!